python/src/sdf/pyJoint.cc

```diff
@@ -97,6 +98,22 @@ void defineJoint(pybind11::object module)
     .def("semantic_pose", &sdf::Joint::SemanticPose,
          "Get SemanticPose object of this object to aid in resolving "
          "poses.")
+    .def("sensor_count", &sdf::Joint::SensorCount,
+         "Get the number of sensors.")
+    .def("sensor_by_index", &sdf::Joint::SensorByIndex,
```

Comment:
This made me realize we don't have mutable versions of `Joint::SensorByIndex` and `Joint::SensorByName`.